src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -56,6 +62,21 @@ public static function getName(): string
      */
     public function __construct()
     {
+        if (empty(System::getEnv('_APP_SMS_PROVIDER')) || empty(System::getEnv('_APP_SMS_FROM'))) {
+            throw new \Exception('Skipped SMS processing. Missing "_APP_SMS_PROVIDER" or "_APP_SMS_FROM" environment variables.');
+        }
+
+        $providers = System::getEnv('_APP_SMS_PROVIDER', '');
+
+        if (!empty($providers)) {
+            $providers = explode(',', $providers);
+            foreach ($providers as $provider) {
+                $this->dsns[] = new DSN($provider);
+            }
+        }
```

Comment:
This should still be in `sendInternalSMSMessage`, otherwise it will stop the messaging service from working unless these vars are set, which may not be the case for self-hosted

Reply:
Updated thanks 👍